mock-core-configs/etc/mock/amazonlinux-2023-aarch64.cfg

```diff
@@ -0,0 +1,5 @@
+include('templates/amazonlinux-2023.tpl')
+
+config_opts['root'] = 'amzn2023'
+config_opts['target_arch'] = 'aarch64'
+config_opts['legal_host_arches'] = ('aarch64',)
```

Comment:
I'm not sure what's up with this file vs the x86_64 version, but it's missing something here and making GitHub angry...

Reply:
Was missing a newline at end of file, so have fixed that just in case that was it.

Reply:
Seems like it was.